spec/factories/digital_object_imports.rb

```diff
@@ -5,6 +5,7 @@
     index { 34 }
     digital_object_data do
       {
+        'assign_uid' => '2f4e2917-26f5-4d8f-968c-a4015b10e50f',
```

Comment:
Rather than adding assign_uid to the base factory definition, it would be better to create a new trait like `:item_with_assign_uid` since lots of tests use this factory or the traits that inherit from it, and with this change those tests are no longer covering the default (and most common) auto-assign-new-uid case.